webapp/javascript/components/AppSelector/SpyNameIcons.tsx

```diff
@@ -0,0 +1,454 @@
+ovy6jREvgUEhR+9zGZqtuUKPNKkDxki1ZwSMEe9ShzqOxngMGb9r7fd/z5G6EfLFirdpsPCD9Nm8
+X4zwGJrzpojQpA4YI8UH1/CqFTcBVhlH2DI+6RSYu1X1hmdpUgcy1nKKNQ0kXvMTYJ8r8iERsTjn
+aM2y3ZDjD+2/oNlrcFak2F4jmsbPo95WsxeBzuO0s+NadvpumXVKqV4CuU1Tba+hpgnkUW8rCR+R
+aYzuArUuhrhl3abIF6tOf+gFQxdJx5c7xqbA6zC7C9S6GZK316ovFsHyKkLp+M8VKJtb0zzFIYme
+kKa9pvnXwHlU7F9jLbCOBChSoWyOpnmrZ5Ktab+EjDLttfIDR6Qmllg6fvDV/yJG0OzvCIYs/5ra
+j4S016qXtK5BOHTBq4rbQc/Xmuv4ZMXrHi/PkwzZMVoaRaR/7b9xCg1IzRDRC1RBLdI0KL86h/Qn
+sy450pY08txafe4sVszgsSBH20qWNhsbaJdNa0mjZW2BXCz1SY+IGHHTqB2SZPOP6CCtkN3R0AwZ
+0cLdYAui/jpxEhmu4CUzhSSBNpIrdu6sWSKDtkv7Hvkq1eeNXXjF0H1xjlSQBPts/vHsfWKXRkt+
+AKu+aseoC28PeiBqjtokBTPcG5dJgq7ec3UetRL6aL1/yNfpuA6CVTDm33zx4tCvKGGtkD9rdnCz
+M7Dtk1kKR2SIX7lj1ARWLOyhsMk2sfv8K3mCvkAWsTt/bE/dSuj7vf00zhHpGGmhaBAb/TNJipos
+He/8SVKGoCP0qdUE0OqP5Oqtim7UyL1+oGZgVKDFQoU2a9AuHT96IhF1WycI2rxnT926KVtzPT6K
+OJCk76jevW6hFxv+C18cNfH4cP/Rkz/92fWugz9/tbN/DPcss56m9VfvuaZOdtxooQiVjQyHaCpl
+iBfm88+RLVNtWHYNOfX+0iVB9z1Tt6/te9NEgCBVwqGIUpc/OEcEEOdgEapYG9dwigBJIPdBE0WU
+5k7+7BEdSGJAK5J/B+nUaveQOg8+M+oOWZbCoghvMyu/9suL0Tql1a/tXn+JjKtYmgidIlqdUFkf
+Ul6MaGEO+zKIuQOZJT42UimHTBGtxuOW1F6ECyQQRzkjWyMo+gOZ3mcYQ8Nj0zMhUkRtpaLSaKOn
+cBU2Revtpzaq1cvLhgA0UkTtEXakTNko55QzghRVa7c7TBGVI1XKdkjxu4vYCf8SQlGTpFmDA60U
+wapPSpXtv6mzQo+AUIoaJK1y1K1DFClY2XZZydsX+RQtO3744m0WS3op6mV884tgLhKnAdiMDEVN
+luY+prxJS/kQB8xZbD0NwNALtnJgnX6ZFDWwsLi+gRgmvRX5Bznz4FVPoeKQf8A9Miloi0ieyP2J
+nniRhQJ3Ir7W/12hKyRTviiCWzi9FIl0vZImafeFYIhtjE3R/9zAKXqbeI2WwKwNsY4XUiX3hQky
+SC+bdB2Hz67duIpoGvkWvS0LRDtebD0VFKXDr6VSIysoRT+xKGrgyiZJ0SL5Eg1HjQ5SErN59pTr
+A+zuSyePVDCKzG2s6V6uESx9QL5Daxl1yX4OD54y9iXb8vwQHJGVMRwPfOn+yudV4l8daG2fItI1
+1YutZ/uHQJp2H+8fcY9neyj9kCuQovNQwho03bu2stlMjSD9bE0HICYGDF84ePDs6739503sfX30
+QOjw+rUUrSGiva7ZgVlGW/alu/9Lfv5bWinqNcJCa9WhOKrDpg9JhvfxnFmQMZvTSlFYBGVMi0pp
+iGTuRexDIvOvBcsWa7XWvO2HKmTtYvmU+NRQ6zzR8hSt01iUInAPXe8mlr/9UIKc6ytpMbym3SHb
+sqP3Y8Eda72mKISmuw0R8ubZ0ThqKJuV9mBlhmAMwdLXGhKwXFDRcI+DTI78Ump71grhG2I32CZh
+5Wu9TeVUNrfEcQJp2UHlaPkTuP/gLWea9Yy5/cgWgvdQzeKLDb3N7zs1l+eIpIMi3Xe0btBY24/W
+5AI2ms1SV2NGK+QmSdSkYoQhzT1AqdsPe3LJ0777FmZZ7gqrXfR5a5ta5muZbmONdNxDJpfO+uAn
+c4Ljz4k01STi/2Wk87fRo7nnHrqgwV9fWpSyOf4qM8h9p+fm5+T4MPqQ7gagQgS1kBBn6UxebEOQ
+5HM0PFUZLzVQGR2hPKK5fzyy/WDZj8FclusjZLI5ccFPKXBcNVsisP3Icn/+RO6116nScyKXkPxN
+5aNV5Ig1M0QsK3yCTAwW8qey2TOmSGXOvP7aiXx60JfE85WNLbTa2+y6rbUwQWoRTNl0q5k7ntYh
+gpoI0LxefzvrVAQI6grgwr+mf2yJKBDUIsmXIPWG0DY+Fw2CmvChbWEw1IynRYOgBpJ5SYZOhzKs
+vsgQ1CJJyiTlwhmU1jbHvkgSVbeeKP224SIlZJMy+TDMUHSROM3ZCGZOvdwEtVBgKNzLLkEOEmhU
+4cypl9cGoWjslk9kX28xlTmTPZErBJGf/wMusCGpjYQ63QAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAy
+Mi0xMi0wOFQxMzo1NzoxNSswMTowMNVsf1UAAAAldEVYdGRhdGU6bW9kaWZ5ADIwMjItMTItMDhU
+MTM6NTc6MTUrMDE6MDCkMcfpAAAAEXRFWHRleGlmOkNvbG9yU3BhY2UAMQ+bAkkAAAASdEVYdGV4
+aWY6RXhpZk9mZnNldAA5MFmM3psAAAAYdEVYdGV4aWY6UGl4ZWxYRGltZW5zaW9uADg1MDSUj6cA
+AAAYdEVYdGV4aWY6UGl4ZWxZRGltZW5zaW9uADI5NuHaAT4AAAAASUVORK5CYII="
+      />
+    </svg>
+  );
+};
+
+export const SPY_NAMES_ICONS: { [k: string]: ReactNode } = {
```

Comment:
Can you improve the typing here so that, once we support a new spyName, we also updated it here?

Reply:
https://github.com/pyroscope-io/pyroscope/blob/dcfd695e0fc9daf425b31b6d7771a69f7463e74f/packages/pyroscope-models/src/spyName.ts#L3-L15

Reply:
ye, sure